models/articles.model.js

```diff
@@ -56,3 +56,18 @@ exports.selectCommentsByArticleId = async (id) => {
 
   return comments;
 };
+
+exports.insertComment = async (body, username, article_id) => {
+  console.log(body, username, article_id);
+  const { rows: article } = await db.query(
```

Comment:
Why do you need to make a request to fetch all the articles? This query seems redundant.